hack/e2e_collect.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Fetch graph data from Cincinnati
+# Input params:
+# * 1 - cincinnati URL
+# * 2 - prefix
+
+type -f curl > /dev/null || {
+  echo Please install curl
+  exit 1
+}
+
+test -z "${1}" && echo "Usage: ./e2e-collect.sh CINCINNATI_URL <tempfile prefix>" && exit 1
+
+TMPFILETEMPLATE="tmp.${2}XXXX.json"
+TMPFILE="$(mktemp -u --tmpdir=. -u ${TMPFILETEMPLATE})"
```

Comment:
Why the usage of a template and the CWD?

Reply:
Might be useful for local testing - e.g. prefix stage and prod outputs separately

Reply:
jwouldn't that be as easy as assigning the output of this script to a variable in the outer scope? e.g.

```shell
GRAPH_STAGE=$(./e2e_collect.sh https://api.openshift.com:443/api/upgrades_info/v1/graph?channel='prerelease-4.1'\&arch='amd64')
GRAPH_PROD=$(./e2e_collect.sh https://api.stage.openshift.com:443/api/upgrades_info/v1/graph?channel='prerelease-4.1'\&arch='amd64')
```

Reply:
Initially I didn't want to output to stdout for comparison (as graphs may be large), but it seems its more intuitive.

Note that `GRAPH_STAGE` and `GRAPH_PROD` still have to saved into a file for comparison via `e2e_compare`, but in that case the user gets to choose the file names

Reply:
> Initially I didn't want to output to stdout for comparison

I'm not suggesting to do that. The variables would contain filenames in my suggestion.

Reply:
> the user gets to choose the file names

I don't think we need that if the path is stored in a variable. I think we should clean them up afterwards though, so maybe we need a cohesive script which does the getting and the comparing and cleans up unconditionally.

Reply:
Ah, I see. I wanted to have a temp file template so that stage / prod results for several tests (e.g. multiarch / channels) would be easy to distinct. This is probably too complex, removed this part